src/commons/utils/corio_utils.py

```diff
@@ -284,19 +283,35 @@ def store_logs_to_nfs_local_server():
         shutil.rmtree(DATA_DIR_PATH)
 
 
-def is_package_installed_local(package_name):
```

Comment:
This method is not needed?
Where this logic is shifted?

Reply:
This is part of install package method.